modules/solid_mechanics/src/materials/HillCreepStressUpdate.C

```diff
@@ -310,5 +329,85 @@ HillCreepStressUpdateTempl<is_ad>::computeStressFinalize(
     this->_max_integration_error_time_step = std::numeric_limits<Real>::max();
 }
 
+template <bool is_ad>
+void
+HillCreepStressUpdateTempl<is_ad>::computeQsigmaChanged(
+    GenericReal<is_ad> & qsigma_changed,
+    const GenericDenseVector<is_ad> & stress_new,
+    const GenericReal<is_ad> & delta_gamma,
+    GenericRankTwoTensor<is_ad> & stress_changed)
+{
+  GenericReal<is_ad> qsigma_square;
+
+  // Hill constants, some constraints apply
+  const Real & F = _hill_constants[_qp][0];
+  const Real & G = _hill_constants[_qp][1];
+  const Real & H = _hill_constants[_qp][2];
+  const Real & L = _hill_constants[_qp][3];
+  const Real & M = _hill_constants[_qp][4];
+  const Real & N = _hill_constants[_qp][5];
+
+  if (!this->_use_transformation)
```

Comment:
What's the purpose of `_use_transformation`?  For testing?  If the two approaches are equivalent, can one be removed?

Reply:
The `_use_transformation` is used for switching to the rotated version of the Hill tensor for cases such as rigid body motion or large deformation (defined in `GeneralizedRadialReturnStressUpdate`). So the approach with `_use_transformation` will yield a different value of qsigma in such cases.